tasks/gen/templates/resource/actions/{{folder_name}}/delete.cr.ecr

```diff
@@ -1,4 +1,4 @@
-class <%= pluralized_name %>::Delete < BrowserAction
+class <%= pluralized_name %>::DeleteOperation < BrowserAction
```

Comment:
This should be reverted. `class <%= pluralized_name %>::Delete` is the correct action name.